Jenkinsfile-smoke.groovy

```diff
@@ -10,7 +10,7 @@ if (env.CHANGE_ID) {
         ocDeployerBuilderPath: "catalog/catalog-api",
         ocDeployerComponentPath: "catalog/catalog-api",
         ocDeployerServiceSets: "catalog,topological-inventory,sources,approval,platform-mq",
-        iqePlugins: ["iqe-self-service-portal-plugin"],
+        iqePlugins: ["self_service_portal"],
```

Comment:
@brahmanim I saw this name the other day on slack, maybe not for this PR, but what would it take to update the name to reflect the current project name?

Reply:
@gmcculloug I am aware of the old name that we are using on our testing plugin.
It's pretty complicated to change this plugin name on all the place that it appears,
GitLab, IQE framework, Vortex, Smoke test .....
There is a thought to change all plugin names to be with the same naming convention,
So I might use this plan to change the plugin name(if it won't be changed again until then).